data/ai/lua/ca_recruit_rushers.lua

```diff
@@ -0,0 +1,32 @@
+-- Make the generic_recruit_engine functions work as external CAs
+
+local dummy_engine = { data = {} }
+local ca_castle_switch = wesnoth.require("ai/lua/ca_castle_switch.lua")
```

Comment:
Really wild question. Is it possible to see if the castle switch CA is present before calling its evaluation?
The idea being that if someone includes this CA and not the castle switch one, then the castle_switch should not be affecting this one.
It didn't matter so much when they were all in a single blob, but this CA is more useful on its own if it is possible.

Reply:
Oh, yes, good point!  I did this, together with the check for the passive_leader aspect as per your other comment, in c3e1ae5.